javascript/net/grpc/web/generator/grpc_generator.cc

```diff
@@ -77,6 +79,20 @@ const char* kKeyword[] = {
     "volatile",   "while",        "with",
 };
 
+// Edit the version here prior to release
+const std::string grpc_web_version = "1.3.1";
+
+string GetVersion() { return grpc_web_version; }
```

Comment:
nit: could probably just inline the constant (rather than having an additional function)? :)

Reply:
Seems actually inlining requires C++17 (if that's what you meant.)
```sh
grpc_generator.cc:83:1: warning: inline variables are a C++17 extension [-Wc++17-extensions]
inline static const std::string GRPC_WEB_VERSION = "1.3.1";
```
So probably this is better?:
```c+=
static const std::string GRPC_WEB_VERSION = "1.3.1";
```

Reply:
Ahhh yes.. sorry i didn't mean literally the "inline" keyword.. what you did was exactly what i was thinking :)